README.md

```diff
@@ -292,14 +292,14 @@ Auf diese Weise wird die Anwendung mit einer PostgreSQL-Datenbankmanagementsyste
 
 Die Demodaten enthalten folgende **Benutzer**, ein Passwort wird nicht benötigt:
 
-| Benutzername         | Rolle                            |
-|----------------------|----------------------------------|
-| user                 | User                             |
-| departmentHead       | User & Abteilungsleiter          |
-| secondStageAuthority | User & Freigabe-Verantwortlicher |
-| boss                 | User & Chef                      |
-| office               | User & Office                    |
-| admin                | User & Admin                     |
+| Benutzername                                 | Passwort | Rolle                            |
+|----------------------------------------------|----------|----------------------------------|
+| user@urlaubsverwaltung.cloud                 | secret   | User                             |
```

Comment:
Should we allow login with username `user`, and so on?

Reply:
beim Login steht halt groß "Email" - daher bin ich da drauf gegangen.